packages/plugin-core/src/commands/ReloadIndex.ts

```diff
@@ -60,10 +61,20 @@ export class ReloadIndexCommand extends BasicCommand<
     // If it already exists, nothing to do
     if (await fs.pathExists(rootSchemaPath)) return;
 
-    const schema = SchemaUtils.createRootModule({ vault });
-    this.L.info({ ctx, vaultDir, msg: "creating root schema" });
-    await schemaModuleOpts2File(schema, vaultDir, "root");
-    return AutoFixAction.CREATE_ROOT_SCHEMA;
+    try {
+      const schema = SchemaUtils.createRootModule({ vault });
+      this.L.info({ ctx, vaultDir, msg: "creating root schema" });
+      await schemaModuleOpts2File(schema, vaultDir, "root");
+      return AutoFixAction.CREATE_ROOT_SCHEMA;
+    } catch (err) {
+      this.L.info({
```

Comment:
should this be error level?

Reply:
This is expected if there are missing vaults. Should I log that at an error level?

Reply:
Maybe keep at info for now.  Besides missing vaults, are there any other potential unexpected errors that may come out of the try block?